src/components/IncorporationAgreement/AgreementType.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <div>
+    <div id="agreement-summary" v-if="isSummary">
+      <v-card flat>
+        <!-- Summary Header -->
+        <div class="agreement-summary-header">
+            <v-icon>mdi-handshake</v-icon>
+            <label class="agreement-summary-title"><strong>Incorporation Agreement and Benefit </strong></label>
+        </div>
+
+        <!-- Summary Warning -->
+        <div v-if="showErrorSummary" class="agreement-invalid-message">
+            <span>
+            <v-icon color="#1976d2">mdi-information-outline</v-icon>
+            This step is not complete.
+            <router-link id="router-link" :to="{ path: '/incorporation-agreement' }">
+                Return to this step to complete it.
+            </router-link>
+            </span>
+        </div>
+        <!-- Summary Content -->
+        <div v-else class="summary-desc">
+            <div><v-icon color="green" class="agreement-valid-icon">mdi-check</v-icon></div>
+             <div>
+                {{ selectedAgreementDescription }}
+            </div>
+        </div>
+      </v-card>
+    </div>
+    <div v-else style="margin-left:1rem;">
```

Comment:
To prevent inline styles, you can use class="ml-4" instead.

https://vuetifyjs.com/en/styles/spacing/